misc-scripts/ontology/sql/patch_95_96_b.sql

```diff
@@ -0,0 +1,26 @@
+-- Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+-- Copyright [2016-2018] EMBL-European Bioinformatics Institute
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+-- patch_95_96_a.sql
+--
+-- Title: Update schema version.
+--
+-- Description:
+--  Added fields to Term and Ontology tables
+
+-- Added columns
+ALTER TABLE ontology ADD COLUMN `title` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL;
+
+ALTER TABLE term ADD COLUMN `iri` text NULL;
```

Comment:
Did some testing, and MySQL can update the charset from latin-1 to utf-8 without clobbering data in existing fields. It would be better to update tables to utf-8 in the patch to keep the schemas consistent, and facilitate adding data with extended-charset characters into a patched schema.

Reply:
beware UTF8 charset, it can make indexes very bloated as each character is 8 bytes long

Reply:
For the `iri`, is not meant to be indexed. For `ontology`, as I said, only 50 lines, index won't grow that much.
Well, the last commit changed the character set on required tables, this might have an impact especially for term table (but still, the table is not that huge, less than 500,000)